app.js

```diff
@@ -1,26 +1,25 @@
 const express = require("express");
-const { getTopics } = require("./controllers/controller");
+const { getTopics, getArticleById } = require("./controllers/controller");
 
 const app = express();
 
 app.get('/api/topics', getTopics);
 
+app.get('/api/articles/:article_id', getArticleById);
+
 app.use(express.json());
 
 app.use("*", (req, res) => {
   res.status(404).send({ msg: "Invalid Path" });
 });
 
 app.use((err, req, res, next) => {
-    if (typeof err === "string") {
-      res.status(400).send({ msg: err });
-    } else next(err);
+  res.status(400).send({ msg: "Invalid input" });
```

Comment:
Make this dynamic: make use of your `err` variable in here and send the `err.msg`

Reply:
Fixed